ui/helpers/utils/display-critical-error.ts

```diff
@@ -24,26 +24,51 @@ export enum CriticalErrorTranslationKey {
  * @throws {ErrorLike} Throws the error after displaying the message.
  * @returns A promise that resolves to never, as it always throws an error.
  */
-export async function displayCriticalError(
```

Comment:
I could have left the name alone, but I thought it was too ambiguous to have both `displayCriticalError` and `displayCriticalErrorPage`. I renamed it to make it a bit more clear that the purpose and parameters were different.